### ISSUE TITLE
[Accessibility: Resize text] Unable to manage guest storage when zoom 200%

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#893]
![5bdf0719b557bf6d](https://cloud.githubusercontent.com/assets/4921718/14019525/0205ccdc-f1b2-11e5-84a6-00d9a7a3c483.jpg)
